firmware/projects/Demo/CAN/Receive/platforms/cli/bindings.cc

```diff
@@ -12,17 +12,17 @@
 namespace mcal {
```

Comment:
Again, we should get this all working with the `stm32f767` platform before merging.

Reply:
Ah that makes more sense